react/EXPERIMENTAL_Table.ts

```diff
@@ -1,3 +1,5 @@
 import Table from './components/EXPERIMENTAL_Table/index'
+import { DENSITIES } from './components/EXPERIMENTAL_Table/constants'
```

Comment:
Is it required to import/export this here?

Reply:
This allows the developer to access these values so that different densities are treated consistently throughout the code.